DigitalLearningSolutions.Data.Tests/DataServices/LearningLogItemsDataServiceTests.cs

```diff
@@ -41,14 +41,20 @@ public void InsertLearningLogItem_inserts_expected_record()
             try
             {
                 // Given
-                const int competencyLearningResourceId = 1;
+                const int learningResourceReferenceId = 1;
                 const int delegateId = 2;
                 const string resourceName = "Activity";
                 const string resourceLink = "www.test.com";
                 var addedDate = new DateTime(2021, 11, 1);
+                competencyLearningResourcesTestHelper.InsertLearningResourceReference(
+                    learningResourceReferenceId,
+                    learningResourceReferenceId,

```

Comment:
Minor, there are a few places in this file where this variable is used as both the table ID and Ref ID, which is fine logic-wise. For readability sake, each should probably be named differently. Just so someone doesn't see this later on, do a double-take, and try and figure out if this is an error.